src/cpp/dynamic-types/TypeObjectHashId.cpp

```diff
@@ -200,63 +211,80 @@ EquivalenceHash& TypeObjectHashId::hash()
     return m_hash;
 }
 
-// TODO(Ricardo) Review
-size_t TypeObjectHashId::getCdrSerializedSize(const TypeObjectHashId& data, size_t current_alignment)
+} // namespace types
+} // namespace fastrtps
+} // namespace eprosima
+
+namespace eprosima {
```

Comment:
Find attached the generated code from [`TypeObjectHashId.idl`](https://github.com/eProsima/Fast-DDS/blob/bugfix/type-object/builtin-annotations/include/fastrtps/types/TypeObjectHashId.idl)
[TypeObjectHashIdCdrAux.txt](https://github.com/eProsima/Fast-DDS/files/12609545/TypeObjectHashIdCdrAux.txt)

Reply:
We are not sure the current types are equal that the defined one in idl files. I would keep the current code.